simpledotorg/simple-server
# Set up IHCI sms reminder experiments for the next six months

**Story card:** [sc-10787](https://app.shortcut.com/simpledotorg/story/10787/set-up-ihci-sms-reminder-experiments-for-the-next-six-months)

## Because

We send out sms reminders when patients miss their BP appointments in three Indian states every month. Since the sms contents remain the same, we're setting these up for the next six months at once.

## This addresses

This PR adds a data migration to create current patient experiments to send notifications every 3 and 7 days from enrollment date, and stale patient experiments to send notifications every 0 and 7 days from enrollment date for the months July through December 2023.

## Test instructions

Comment out the line doing country and production environment checks, and run `bundle exec rails db:migrate:with_data `. All the transactions creating experiments should run without errors.
